src/heartratetask/HeartRateTask.cpp

```diff
@@ -29,7 +29,7 @@ void HeartRateTask::Work() {
     uint32_t delay;
     if (state == States::Running) {
       if (measurementStarted)
-        delay = 40;
+        delay = 100;
       else
         delay = 100;
```

Comment:
This if-statement seems unnecessary...

Reply:
Thanks! Fixed the magic number by using a public constant in the Ppg class.

Reply:
Cool! My point was more that no matter whether `measurementStarted` is true or false, `delay` is always set to 100, so the if-statement could be removed.

Reply:
Understood. Since it is possible to change the acquisition frequency in the future I thought it best to make it so there is just a single value to change. Now there is one constexpr (deltaTms) to change in Ppg.h and everything else is calculated from that. I used to have to change two things in order to update the acquisition rate. Thanks again for pointing that out, I had forgot to address that issue!